.rubocop.yml

```diff
@@ -3,3 +3,9 @@ inherit_from:
 
 AllCops:
   TargetRubyVersion: 2.3
+
+Style/TrailingCommaInArrayLiteral:
```

Comment:
I'm not opposed to adding these (seems to be new in the Shopify style guide), but doing so now would cause rebase hell for the several people with huge PRs in flight.